ui/src/main/js/distribution/channels/jira/cloud/JiraCloudDistributionConfiguration.js

```diff
@@ -0,0 +1,118 @@
+                errorName={FieldModelUtilities.createFieldModelErrorKey(JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.project)}
+                errorValue={errors[JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.project]}
+            />
+            <TextInput
+                id={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.issueType}
+                label="Issue Type"
+                description="The issue type to open when creating an issue in Jira Cloud."
+                name={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.issueType}
+                readOnly={readonly}
+                required
+                onChange={FieldModelUtilities.handleChange(data, setData)}
+                value={FieldModelUtilities.getFieldModelSingleValueOrDefault(data, JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.issueType)}
+                errorName={FieldModelUtilities.createFieldModelErrorKey(JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.issueType)}
+                errorValue={errors[JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.issueType]}
+            />
+            <TextInput
+                id={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.resolveWorkflow}
+                label="Resolve Transition"
+                description="If a transition is listed (case sensitive), it will be used when resolving an issue. This will happen when Alert receives a DELETE operation from a provider. Note: This must be in the 'Done' status category."
+                name={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.resolveWorkflow}
+                readOnly={readonly}
+                onChange={FieldModelUtilities.handleChange(data, setData)}
+                value={FieldModelUtilities.getFieldModelSingleValueOrDefault(data, JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.resolveWorkflow)}
+                errorName={FieldModelUtilities.createFieldModelErrorKey(JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.resolveWorkflow)}
+                errorValue={errors[JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.resolveWorkflow]}
+            />
+            <TextInput
+                id={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow}
+                label="Re-open Transition"
+                description="If a transition is listed (case sensitive), it will be used when re-opening an issue. This will happen when Alert receives an ADD/UPDATE operation from a provider. Note: This must be in the 'To Do' status category."
+                name={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow}
+                readOnly={readonly}
+                onChange={FieldModelUtilities.handleChange(data, setData)}
+                value={FieldModelUtilities.getFieldModelSingleValueOrDefault(data, JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow)}
+                errorName={FieldModelUtilities.createFieldModelErrorKey(JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow)}
+                errorValue={errors[JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow]}
+            />
+            {!readonly && (
```

Comment:
Jira cloud here seems to hide the collapsiblePane where Jira server doesn't. Perhaps this check should be removed?